packages/Liquid-Core.package/LiquidPoll.class/instance/closeWithPassword..st

```diff
@@ -0,0 +1,5 @@
+accessing
```

Comment:
[Pebble] You need to go over all the methods you added and classify them correctly, since Squeak automatically classifies them in the currently selected category